en/components/calendar.md

```diff
@@ -74,11 +74,11 @@ Notice that the calendar header is not rendered when the selection is either `mu
 
 #### Localization and formatting
 
-Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbase.html#formatviews).
+Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbasedirective.html#formatviews).
```

Comment:
Hold on, why are those links going to the base classes?!
_(Technically should be igxcalendarbase**directive** now, but then again that's not exported and has no place in the API and I marked it hidden.)_

All those links **should** go to the actual component people will interact with:
```suggestion
Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#formatviews).
```

Reply:
Because they were going there previously :)

But I don't mind applying your suggestion ofc, it makes a lot of sense

Reply:
They seem to work now though.

https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/igxcalendarbase.html#selection

Reply:
I'll rename igxcalendarbase to igxcalendarcomponent but that will probably not be all links pointing to base classes in the docs.

Reply:
It makes sense for the grids to use base API in templates for simplicity (though not optimal), but not the single calendar.
And those work now, but like I said I marked it hidden so it should drop out of the API files.